### ISSUE TITLE
[Docs Infra] Remove "<<<" from code blocks

### ISSUE THREAD

**Issue by angelinalg** [ray-project/ray#34439]
### Description

CSAT feedback from Feb 11:
"delete ">>>" from Example"

https://analytics.google.com/analytics/web/#/report/content-event-events/a154591315w164762454p165427173/_u.date00=20221001&_u.date01=20230413&explorer-table.plotKeys=%5B%5D&explorer-table.secSegmentId=analytics.pagePath&_r.drilldown=analytics.eventCategory:RateTheDocs,analytics.eventAction:Suggestion,analytics.eventLabel:delete%20%22%3E%3E%3E%22%20from%20Example

### Link

https://docs.ray.io/en/master/rllib/rllib-algorithms.html

**Comment by angelinalg** [comment]
cc: @richardliaw @bveeramani

**Comment by bveeramani** [comment]
Yeah, we should absolutely remove REPL prompts (`>>>`) from longer examples like these.

**Comment by cszhu** [comment]
This P2 issue has seen no activity in the past 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please comment and remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
